src/megatron/energon/epathlib/epath.py

```diff
@@ -74,6 +75,24 @@ def __init__(
             else:
                 raise ValueError(f"Unknown protocol: {protocol}")
 
+    @staticmethod
+    def _resolve(path: Union[str, PurePosixPath]) -> PurePosixPath:
```

Comment:
Can you add a docstring? I guess all that this does is getting rid of `..` and `.`?

Reply:
Done